### ISSUE TITLE
accessibility: optimize ion-searchbar for the aria-label values Search text form and Reset button

### ISSUE THREAD

**Issue by terrancle** [ionic-team/ionic#21326]
In ion-searchbar, it is not currently possible to define the text entered in the resetButton, as well as the label of the search field.
By default, the aria-label attribute for the name of the text field is "Search text" and "Reset" for the Reset button which allows you to empty the field.
This lack is a problem for VoiceOver, when a translation has to be done in several languages.
Could you add an attribute for resetButtonText and for searchText, as you did very well for cancelButtonText?
In this way, the aria-label will be generated with the value indicated in the ion-searchbar tag in each project, depending on the language of the user.

**Comment by liamdebeasi** [comment]
Thanks for the issue. This seems to be a recurring issue with certain Ionic components, so I created a new issue that we can use to track this. I am going to close this issue in favor of https://github.com/ionic-team/ionic/issues/21327.
